SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3531,6 +3540,64 @@ bool USpatialNetDriver::HasTimedOut(const float Interval, uint64& TimeStamp)
 	return false;
 }
 
+void USpatialNetDriver::ServerReplicateActors_BuildConsiderList(TArray<FNetworkObjectInfo*>& OutConsiderList, const float ServerTickTime)
+{
+	const bool bSpatialNetworking = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
+
+	if (!bSpatialNetworking || AuthorityDebugger == nullptr)
+	{
+		// Default case to only consider auth actors for replication
+		Super::ServerReplicateActors_BuildConsiderList(OutConsiderList, ServerTickTime);
```

Comment:
I believe this will return "auth" and "non-auth" actors which is not what we want?

Reply:
Maybe it would be easier to understand, but not as optimal, if this function:
a. Called Super, getting list that includes auth and non-auth actors
b. (OPTIONALLY) Loop through this list, looking for non-auth actors and checking them
c. Use RemoveAll (or swap would be better) to prune the list to auth actors

I don't know how many things are in this list, or how often its called, so I don't know the magnitude of perf impact this might have.